docs/users-guide/09-multi-series-charting.md

```diff
@@ -7,73 +7,79 @@ Data in isolation is rarely all that useful. One of the best ways to add context
 
 ### Combining Number charts
 
-If you need to compare single numbers and get a sense of how they differ, Metabase also lets you turn multiple Number charts into a bar chart. To do this, follow the same process outlined above. While editing a dashboard, click “edit data” on the Number chart of your choice and then select the other saved question(s) you’d like to see represented on the bar chart. (At Metabase, we use this to create simple funnel visualizations.)
+If you need to compare single numbers to get a sense of how they differ, Metabase can turn multiple number charts into a bar chart. 
 
-### Creating a multi-series visualization in the query builder.
+As above, while editing a dashboard, hover over a number chart of your choice, and click on the icon with the **+** and bar chart icon to add a saved question. 
 
-If you’re creating a new question you can also view the result as a multi-series visualization. To do this you’ll need to summarize and add two groupings to your question.
+![Add bar chart](images/multi-series-charts/add_bar_chart.png)
 
-As an example, we might want to see which website or service is referring the most people to our website. (In the sample dataset that ships with Metabase this would involve using the `Source` and `Created At` columns of the `People` table.)
+Use the search bar to find other saved question(s) that you’d like to see represented on the bar chart, and click the checkbox to add them to your chart. In this case, we added **Widget orders** to compare them to **Gadget orders**.
 
-To do this we’d click the Summarize button, and then add `Source` and `Created At` as groupings (the `count of rows` metric that we want is already selected by default). [Learn more about asking questions](04-asking-questions.md).
+![From numbers to bar chart](images/multi-series-charts/numbers_to_bar_chart.png)
 
-Metabase will automatically display a multi-series line chart visualization of how each referrer has performed for us.
+### Multi-series charts, values, and legibility
 
-![multiseriesquerybuilder](images/MultiSeriesQueryBuilder.png)
+When displaying multiple series, it’s important to keep legibility in mind. Combining many series can sometimes decrease the communication value of the data.
 
-It’s worth noting that at this time you won’t be able to add another saved question to multi-series visualizations made in this fashion. Metabase can only visualize up to 20 values of a dimension at once, so you may need to filter the values if you're selecting a field that contains a lot of values like "State."
+Metabase allows you to add values to multi-series charts, but go easy on this feature, especially on charts with lots of data points. Adding values to multiple series, each with many data points, can make charts more difficult to read.
```

Comment:
Worth noting here is that you can turn values on for just _one_ series (or a few series), by expanding that series' details in the Display tab, and turning on the display values toggle for it. You can also toggle on values for the whole chart, then turn it off selectively for individual series.